### PR TITLE
ES6 support + fixed controls, album, lyrics...

### PR DESCRIPTION
**Pull request by lodi-g**
Fixed the extension (mostly just selectors were wrong).

Also changed the logic to query lyrics, unfortunately the old API was uh quite Brazilian only. I think Genius.com is larger and offers more diversity.

Implemented a quick & dirty backend here: https://gitlab.com/lodi-g/spotify-control-chrome-extension-backend (using gitlab for ci/cd) to protect the accessToken and avoid spamming the api.

Lmk if needs anything.

### CODE REVIEW

**Comment by lodi-g** [comment]
Fixes #7 #8 #9

File: js/main.js [hunk 1]
```diff
@@ -1,330 +1,388 @@
...
+    headers: {
+      'Content-Type': 'application/json',
+    },
+  });
+
+  if (res.status === 404) {
+    updateLyricsText('Song not found on genius.com');
+  } else if (!res.ok) {
+    updateLyricsText(
+      'Unknown error, feel free to open an issue on GitHub: https://github.com/lodi-g/spotify-control-chrome-extension',
```
Comment (thiagoribeir015) [inline js/main.js:212]: hey, I'm glad to get this merged and then release a new version of the extension asap... can you please update the references from your github to mine across your changes? I'd like to keep all issues at the same place.

**Review by thiagoribeir015 — CHANGES_REQUESTED** [review]
Hey, it looks great, thanks a lot for that!

Can you please change the refs to this github project and we'll be good to merge and release a new version :)

**Comment by lodi-g** [comment]
Done!

**Comment by thiagoribeir015** [comment]
hey @lodi-g new version has been published! thanks a lot for collaborating! 4.5k users are happier now :)

**Pull request merged**